src/test/resources/templates/testSubflowLogic/subflowAddItem.html

```diff
@@ -9,8 +9,8 @@
       <main id="content" role="main" class="form-card spacing-above-35">
         <th:block
             th:replace="~{fragments/cardHeader :: cardHeader(header='Subflow Page', subtext='This is a page within a subflow')}"/>
-        <!-- Test that the currentSubflowItem is passed in by the controller: -->
-        <div th:text="${currentSubflowItem.firstNameSubflow}"></div>
```

Comment:
I'm not sure what this was doing exactly but firstNameSubflow was null and I couldn't figure out what would have been setting it in the first place.

Reply:
I wonder if that was debug that did get taken out?

Reply:
Should that `div` even be there?  Not exactly sure what it's doing either.

Reply:
I think you might be right, I'll take it out all together.

Reply:
@spokenbird Did you decide to remove this?